backend/slackbot/handler.js

```diff
@@ -0,0 +1,156 @@
+        };
+    }
+
+    try {
+        await sendStats(statistics);
+    } catch (error) {
+        return {
+            statusCode: 500,
+            error: `Error sending to slack: ${error}`
+        };
+    }
+
+    return {
+        statusCode: 200, // OK
+        body: JSON.stringify({
+            apps: statistics
+        })
+    };
+}
+
+/**
+ *
+ * @param gatewayURL the gateway url for aws lambda endpoint
+ * @returns returns list of ford apps
+ */
+async function getFordApps() {
+    let endpoint = gatewayURL + '/apps';
+    return axios.get(endpoint);
+
+}
+
+/**
+ *
+ * @param fordApps the list of apps we want statistics from
+ * @returns appStats a list of statistics for each specified version of an app
+ */
+async function getFordStatistics(fordApps){
+    let promises = [];  // Array for promises
+    let appStats = [];  // Array for App Statistics for each version of each app we want
+
+    let apps = fordApps.data;
+    let endpoint = gatewayURL + '/stats';
+
+    for (let i in apps){
+        let app = apps[i];
+
+        // Parsing out dates right now, needs to be changed and cleaned
+        let endDate = app.maxDate + "T00:00:00.000Z";
+        let dates = endDate.split('-');
+        let day = parseInt(dates[2]);
+        let month = dates[1];
+        let startDay = day - 7;
+        if (day < 7) {
+            startDay = 31 - day;
+            if (month == '01') {
+                month = '12';
+            }
+        }
+        let startDate = dates[0] + '-' + month + '-' + startDay + 'T00:00:00.000Z';
+
+        // Currently sorting the list of app versions assuming latest
+        let sortedVersions = app.versions;
+        sortedVersions.sort();
+        let latestVersion = sortedVersions[sortedVersions.length-1];
+
+        // Parameters to make request to stats endpoint
+        let params = {
+            "appIdStore": i,
+            "version": latestVersion,
+            "startDate": startDate,
+            "endDate": endDate,
+            "stats": [{
+                "rawReviews": null
+            }]
+        };
+
+        // Push promise to array of promises
+        promises.push(axios.post(endpoint, params));
+        // Create the dictionary for this specific version of either app
+        appStats.push({
+            "appIdStore": i,
+            "version": latestVersion,
+            "statistics": []
+        })
+    }
+
+    // Await all promises
+    let allAppsPromises = Promise.all(promises);
+    let allApps = [];
+
+    try {
+        allApps = await allAppsPromises;
+    } catch (error) {
+        throw error;
+    }
+
+    // Add statistics to the list of apps
+    let appStatistics = [].concat(allApps);
+    for (let i = 0; i < appStatistics.length; i++) {
+        appStats[i].statistics = appStatistics[i].data.stats;
+    }
+
+    return appStats;
+}
+
+/**
+ * @param statistics list of statistics to output to slack
+ */
+async function sendStats(statistics){
+    let message = 'Hello World';
```

Comment:
Are we not sending any stats to Slack yet? I'd like to at least send the raw data over if possible - that shows a lot more progress from the user's perspective than just a Hello, World message.